src/ServicePulse.Host/vue/src/composables/serviceServiceControlUrls.ts

```diff
@@ -1,29 +1,36 @@
-import { ref } from "vue";
+import { type Ref, ref } from "vue";
 
-export const serviceControlUrl = ref(null);
-export const monitoringUrl = ref(null);
+export const serviceControlUrl = ref<string>();
+export const monitoringUrl = ref<string>();
 
 export function useIsMonitoringDisabled() {
-  return monitoringUrl.value === null || monitoringUrl.value === "" || monitoringUrl.value === "!";
+  return monitoringUrl.value === undefined || monitoringUrl.value === "" || monitoringUrl.value === "!";
```

Comment:
i think we want this first clause to be monitoringUrl.value == null, since it depends on how the form is handled whether the clearing of a value will result in an empty string or null, whereas it starts above as undefined.